ha/core/controllers/pcs/pcs_controller.py

```diff
@@ -72,6 +71,27 @@ def clean_failure_count(self, node_id):
         _output, _err, _rc = self._execute.run_cmd(const.PCS_NODE_CLEANUP.replace("<node>", node_id),
                                                    check_error=False)
 
+    def auth_node(self, node_id, cluster_user, cluster_password):
```

Comment:
convert it to protected function

from `auth_node` to `_auth_node`

Reply:
Done